chart/templates/environment-secrets.yaml

```diff
@@ -44,3 +44,39 @@ spec:
         - regexp:
             source: "[^a-zA-Z0-9 -]"
             target: "_"
+
+---
+apiVersion: external-secrets.io/v1beta1
+kind: SecretStore
+metadata:
+  name: vets-api-checkin-SecretsManager-secretstore
+  annotations:
+    argocd.argoproj.io/hook: PreSync
+    argocd.argoproj.io/sync-wave: "-4"
+spec:
+  provider:
+    aws:
+      service: SecretsManager
+      region: us-gov-west-1
+      auth:
+        jwt:
+          serviceAccountRef:
+            name: vets-api
+
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: checkin-secretsmanager-vars
```

Comment:
use pre-existing

Reply:
We can't use the pre-existing because the secret store we want to use is SecretsManager.